### DISCUSSION TITLE
Learning the scaler per-atom quantities

### DISCUSSION THREAD

**Discussion by xfanak** [deepmodeling/deepmd-kit#836] [Ideas]
Dear DP developers, 

I see that DP can fit the atomic dipole now. Is there an even simplified function that allows learning of scaler per-atom quantities? For example, I want to fit the atomic activation energies, or displacement after some time. For the latter quantity, can I provide the travelled distance along each axis, namely dx, dy, dz, and just use the 'dipole' tag in fitting_net?

Thanks.

**Comment by amcadmus** [comment]
per-atom scaler one can train the model with `atom_ener` labels that are stored in `atom_ener.npy` files. The atomic scaler values of one frame are written on one line and the raw file contains number of frames lines.  One can include the loss of `atom_ener` in the total loss by setting `start_pref_ae` and `limit_pref_ae` to non-zero values. More details please ref to the [documentation of deepmd-kit inputs](https://github.com/deepmodeling/deepmd-kit/blob/devel/doc/train-input-auto.rst)

one can use `dipole` to train the per-atom displacement.

**Reply by xfanak** [reply]
Thanks. I will try.
